lesson9.tex

```diff
@@ -0,0 +1,123 @@
+\documentclass{lessonplan}
+
+% Constants
+\newcommand{\lessonNumber}{}
+\title{Lesson \lessonNumber: The B\&B Stock Exchange Game}
+\author{\linkHome}
+\date{}
+
+% Buildup
+\setLessonNumber{\lessonNumber}
+
+
+%Document
+\begin{document}
+
+  \maketitle
+
+  \section{Overview}
+    \subsection{Summary}
+      This lesson will be a simulation of a stock exchange on a tropical island named Bot Island. 
+      We will teach student what stocks are, how they work, and how they are influenced by real-world events.
+      For more advanced students, it delves into how the stocks make money (through compound interest and dividends)
+      and what are the best strategies for making the most money.
+
+    \subsection{Objectives}
+    \begin{itemize}
+      \item Learn what a stock is, and how to buy and sell stocks
+      \item (Intermediate) Learn about compound interest, and how money grows when you leave it in 
+                an account that gains interest. 
+      \item (Advanced) Learn different strategies of how to handle money regarding the market (such as 
+               diversification, investing in hot-item stocks, avoiding the stock market altogether, etc), and discuss 
+               different kinds of ways to diversify a portfolio (including but not limited to bonds, index funds, 
+               mutual funds, and ETFs). Also learn how to examine and rate a stock by using information like balance sheets P/E ratios.
+      \item (Cooperative) Students can create their own "firms" by grouping up and discussing how they would buy or sell their stocks.
+    \end{itemize}
+    \subsection{Key Vocabulary Terms}
+    \begin{itemize}
+      \item Stocks, buy/sell (in context of the market)
+      \item (Intermediate) Compound Interest
+      \item (Advanced) Portfolio, diversification, bonds, index funds, mutual funds, ETFs, balance sheet, P/E ratios
+    \end{itemize}
+    \subsection{Materials}
+      \begin{itemize}
+        \item Instructor Laptop 
+        \item Pens and paper 
+        \item "BotCash" - papers printed with the B\&B logo to act as currency
+        \item Papers printed as forms to declare buys/sells
+        \item Prizes for students to exchange for their BotCash, ranging from lesser-value items like pencils and erasers to larger-value items
+                 like pencil cases, sketch books, and pen or highlighter sets
+      \end{itemize}
+  \section{Procedure}
+    \subsection{Setup (10 minutes)}
+     Prepare instructor laptop with slideshow containing information about what a stock is, and how you can buy and sell stocks.
+     Another slideshow should follow explaining the premise of Bot Island, and include the fake newspaper screenshots from 
+     Bot Island explaining all the current events that occur on Bot Island. Instructors discuss amongst themselves what
```

Comment:
Also, if we're going to have a PPT as part of the lesson, we should provide the PPT in our repo. We should work on creating one of those as well, maybe with a supplemental materials folder for each lesson.

Reply:
Also, we need to make sure that if we're doing something like this, we make it engaging and basic enough for young kids, especially during the presentation part.